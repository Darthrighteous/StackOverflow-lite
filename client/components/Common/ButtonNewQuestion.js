import React from 'react';
import { Link } from 'react-router-dom';

const ButtonNewQuestion = () => {
  return(
    <Link to="/new-question">
      <button className="btn-new-question" type="button">
        Ask a Question
      </button>
    </Link>
  );
};

export default ButtonNewQuestion;
