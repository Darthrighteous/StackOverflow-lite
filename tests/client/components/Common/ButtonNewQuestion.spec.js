import React from 'react';
import { shallow } from 'enzyme';

import ButtonNewQuestion from '../../../../client/components/Common/ButtonNewQuestion';

test('New Question button snapshot test', () => {
  const btnNewQuestion = shallow(
    <ButtonNewQuestion />
  );
  expect(btnNewQuestion).toMatchSnapshot();
});
