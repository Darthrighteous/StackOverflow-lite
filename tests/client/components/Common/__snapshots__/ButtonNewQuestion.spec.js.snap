// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`New Question button snapshot test 1`] = `
<Link
  replace={false}
  to="/new-question"
>
  <button
    className="btn-new-question"
    type="button"
  >
    Ask a Question
  </button>
</Link>
`;
